dropshots/src/main/java/com/dropbox/dropshots/Dropshots.kt

```diff
@@ -205,19 +296,25 @@ public class Dropshots internal constructor(
     filePath: String?,
     screenshot: Bitmap,
     referenceImage: Bitmap,
-    mask: Mask?
+    mask: Mask?,
   ): String {
     val screenshotFolder = File(rootScreenshotDirectory, "diff".appendPath(filePath))
     val diffImage = generateDiffImage(referenceImage, screenshot, mask)
-    return writeImage(screenshotFolder, name, diffImage)
+    return writeImage(screenshotFolder, name, diffImage, "_diff")
   }
 
-  private fun writeImage(dir: File, name: String, image: Bitmap): String {
+  private fun writeImage(dir: File, name: String, image: Bitmap, nameSuffix: String = ""): String {
     if (!dir.exists() && !dir.mkdirs()) {
       throw IllegalStateException("Unable to create screenshot storage directory.")
     }
 
-    val file = File(dir, "${name.replace(" ", "_")}.png")
+    val file = File(dir, "${name.replace(" ", "_")}$nameSuffix.png")
```

Comment:
Let's ditch the replace here, as well, since that will override whatever the user told us to name the file. Then you can do something like `arrayOf(name, nameSuffix).joinToString("_")` to remove the need for the `_` in the suffix, if you want.